endpoints/logger.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Mandatory exercise 1
+ *
+ * Mandatory exercse 1
+ *
+ * API version: 1.0.0
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+
+package endpoints
+
+import (
+	"log"
+	"net/http"
+	"time"
+)
+
+func Logger(inner http.Handler, name string) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		start := time.Now()
+
+		inner.ServeHTTP(w, r)
+
+		log.Printf(
+			"%s %s %s %s",
+			r.Method,
+			r.RequestURI,
+			name,
+			time.Since(start),
+		)
+	})
+}
```

Comment:
Should the logger and router be in endpoints or just in the root?

Reply:
I agree it is a better option to put these file in the root